src/wild_encounter.c

```diff
@@ -864,10 +880,84 @@ bool8 DoesCurrentMapHaveFishingMons(void)
         return FALSE;
 }
 
+static u32 GetLastFishingSpecies(void)
+{
+    return sLastFishingSpecies;
+}
+
+static bool32 DoesSpeciesMatchLastFishingSpecies(u32 species)
+{
+    return (species == GetLastFishingSpecies());
+}
+
+static u32 GetCurrentChainFishingDexnavStreak(void)
```

Comment:
Apologies for the nitpicking, but we should probably be consistent in `Dexnav` vs `DexNav` (I think the latter is preferable since it's the official capitalisation)

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/4343/commits/90fbd32f58b69519d80635317808972a51595972